packages/app/src/cli/prompts/event/trigger.ts

```diff
@@ -0,0 +1,121 @@
+import {DELIVERY_METHOD} from '../../services/event/trigger.js'
+import {ui} from '@shopify/cli-kit'
+
+export async function topicPrompt(defaultValue?: string): Promise<string> {
+  const input = await ui.prompt([
+    {
+      type: 'input',
+      name: 'topic',
+      message: 'Webhook Topic Name',
+      default: defaultValue === undefined ? '' : defaultValue,
+      validate: (value) => {
+        if (value.length === 0) {
+          return "Topic name can't be empty"
+        }
+        return true
+      },
+    },
+  ])
+
+  return input.topic
+}
+
+export async function apiVersionPrompt(defaultValue?: string): Promise<string> {
+  const input = await ui.prompt([
+    {
+      type: 'input',
+      name: 'apiVersion',
+      message: 'Webhook ApiVersion',
+      default: defaultValue === undefined ? '2022-07' : defaultValue,
```

Comment:
If I'm not wrong, the api version follows the convention `{year}-{month}`. I'd codify that convention into a validation that ensures developers are not passing an invalid api version.